modules/KoreBuild.Tasks/KoreBuild.Tasks.csproj

```diff
@@ -8,6 +8,7 @@
   <ItemGroup>
     <Content Include="*.props" CopyToPublishDirectory="PreserveNewest" />
     <Content Include="*.targets" CopyToPublishDirectory="PreserveNewest" />
+    <Compile Include="..\..\shared\Microsoft.Extensions.CommandLineUtils.Sources\Utilities\*.cs" />
```

Comment:
We have near identical code for this in a couple of repos now, should we just start publishing it? I'm not a fan of the ".\shared\" pattern.

Reply:
It is published as nuget package from aspnet/Common, but because aspnet/BuildTools can't depend on aspnetcore-ci-dev, we made a copy of it in this repo.

Reply:
Ah makes sense.